weaviate/values.yaml

```diff
@@ -513,6 +513,19 @@ modules:
         # enable if running with CUDA support
         # nvidia.com/gpu: 1
 
+  # The qna-openai module uses OpenAI Completions API
+  # to dynamically answer given questions.
+  # More information about OpenAI Completions API can be found here:
+  # https://beta.openai.com/docs/api-reference/completions
+  qna-openai:
+
+    # enable if you want to use OpenAI module
+    enabled: false
+
+    # Set your OpenAI API Key to be passed to Weaviate pod as
+    # an environment variable
+    apiKey: ''
```

Comment:
Could we use the `fail` keyword in case both `qna-openai` and `t2v-openai` have keys set, but the keys aren't identical? Something along the lines of:

```
// TODO: execute code below only if both are set
{{- if ne .Values.qna-openai.apiKey .Values.text2vec-openai.apiKey }}
{{- fail "two OpenAI modules activated, but they keys differ. Use the same key on all activated OpenAI modules." }}
{{- end }}
```

For brevity, I've replaced the first if statement with a comment, but of course we would need that as well. We could probably execute the above snippet as part of the template for the secret where we already do similar checks. WDYT?

Reply:
[This](https://stackoverflow.com/a/61504009/5322199) is where I got the `fail` keyword from :-)

Reply:
Why should the keys be identical?

Reply:
Added check to `weaviateOpenAISecret`